src/Kernel-Tests.package/ClassTest.class/instance/testReferencedClasses.st

```diff
@@ -1,15 +1,9 @@
 testing - navigation
 testReferencedClasses
-	{(RBLintRuleTest
-		->
-			{RBRefactoryTestDataApp.
-			Object.
-			Array.
-			WriteStream.
-			String}). 
-	 (RBBasicLintRuleTest -> {RBSelectorEnvironment}).
-	 (RBCompositeLintRuleTest -> {OrderedCollection})
+	{(ExceptionTester -> { MyTestNotification. Warning. String. MyResumableTestError. OrderedCollection. MyTestError}). 
```

Comment:
I get your change, and It's good enough. But we should think at some point to make the test only depend on an artificial case.